BigBaseV2/write_player_game_state_data_node.cpp

```diff
@@ -0,0 +1,14 @@
+#include "hooking.hpp"
+
+namespace big
+{
+	bool hooks::write_player_game_state_data_node(rage::netObject* player, CPlayerGameStateDataNode* node)
+	{
+		auto ret = g_hooking->m_write_player_game_state_data_node_hook.get_original<decltype(&write_player_game_state_data_node)>()(player, node);
+
+		if (g->spoofing.spoof_hide_god)
+			node->m_is_invincible = false;
```

Comment:
I'd reccomend adding these, as they would prevent potential future methods from being detectable:
```c++
a2->m_is_invincible = false;
a2->m_bullet_proof = false;
a2->m_melee_proof = false;
a2->m_steam_proof = false;
a2->m_explosion_proof = false;
a2->m_water_proof = false;
a2->m_fire_proof = false;
a2->m_collision_proof = false;
```

Reply:
Can be added in a future PR.